tests/auto/tests/test93.js

```diff
@@ -0,0 +1,7 @@
+
+$.get("volData/model1.prmtop",  function (data){
+    var m = viewer.addModel(data, "prmtop");
```

Comment:
you should have to call animate to see it move and should be able to set the style (cartoon would be good, since otherwise you only see water).  Also, need to set the test up so it isn't using a localhost address.  If there is a public mdserv demonstration server, use that, otherwise we can set one up on 3dmol.csb.pitt.edu